### PR TITLE
update ui add template +fixed total num

### PR DESCRIPTION
**Pull request by shir149**

### CODE REVIEW

File: Front/js/scripts.js [hunk 1]
```diff
@@ -111,15 +111,7 @@ function getTotalSentEmail() {
         type: 'GET',
         success: function (totalNum) {
             console.log(totalNum);
```
Comment (peerfikhman) [inline Front/js/scripts.js:113]: We need to delete unnecessary printings

File: Front/js/scripts.js [hunk 2]
```diff
@@ -173,29 +164,28 @@ function getAllTemplates() {
...
             let htmlArray = [];
-            // let templatesArray =[];
-            // templatesArray.push(templates);
-            for (let i = 0; i < templates.length; i++) {
-                localStorage.setItem(i, JSON.stringify(templates[i].template_id));
+        let templateBox = "<li className =`list-group-item list-group-item-primary` ";
+        for (let i = 0; i < templates.length; i++) {
+            let name = templates[i].name;
+            templateBox+= '<li className =`list-group-item list-group-item-primary` '+'(<a onclick=getTemplateByID('+`${[i]}`+')> '+ `${name}` +' </a></li>';
+
```
Comment (peerfikhman) [inline Front/js/scripts.js:176]: Maybe better to create elements rather then to writing the HTML, not sure if

File: Front/js/scripts.js [hunk 3]
```diff
@@ -224,59 +214,29 @@ function getTemplateByID(num) {
...
-    for (let i = 0; i < nameArray.length; i++) {
-        mytable += "<tr>";
 
-        mytable += "<th scope='row'>" +
-            "<button onclick='getTemplateByID(" + `${[i]}` + ")'" + "class=button>Add Template</button>" +
-            "<button onclick='DeleteByID(" + `${[i]}` + ")'" + "class=button>Delete Template</button>" +
-            "<button onclick='edit()'" + "class=button>Edit Template</button>"
-        {/* mytable += "<th scope='row'><div id=list-items><a href=#> Hindi </a><a href=#> English </a><a href=#> Spanish </a> <a href=#> Chinese </a> <a href=#> Japanese </a> </div><button id="+ `${i}`+">" + "</th>"; */
-        }
+function createTemplateTable(nameArray,IdArray,htmlArray){
```
Comment (peerfikhman) [inline Front/js/scripts.js:231]: Not sure if those functions are needed anymore since we have new dynamicly loaded scripts in templatesScript.js

Comment (peerfikhman) [inline Front/js/scripts.js:231]: Again function in templatesScript

File: Front/js/scripts.js [hunk 4]
```diff
@@ -288,15 +248,41 @@ function createTemplateTable(nameArray, IdArray, htmlArray) {
 
 }
 
-function clickTemplate() {
-    $(document).on('click', '#template', function (e) {
-        e.preventDefault();
-        alert("herrre");
+function getTotalnumTemplate() {
```
Comment (peerfikhman) [inline Front/js/scripts.js:251]: Showed be added to templatesScript.js

File: Front/js/scripts.js [hunk 5]
```diff
@@ -288,15 +248,41 @@ function createTemplateTable(nameArray, IdArray, htmlArray) {
...
+    $.ajax({
+        url: 'http://localhost:3000/templates/num',
+        type: 'GET',
+        success: function (totalNum) {
+            console.log(totalNum);
+            document.getElementById("total").innerHTML = totalNum;
+
+
+        },
+        error: function () {
```
Comment (peerfikhman) [inline Front/js/scripts.js:261]: Very Good!!!! error handling we need to implement more of these

**Review by peerfikhman — APPROVED** [review]

**Comment by sufka861** [comment]
resolved the only conflicts were in the DB
please take notice of Peer's notes and edit what is rellevant in branch Develop after the merge

**Pull request merged**